opensearch-operator/pkg/helpers/reconcile-helpers.go

```diff
@@ -22,9 +22,9 @@ func ResolveInitHelperImage(cr *opsterv1.OpenSearchCluster) (result opsterv1.Ima
 	}
 
 	// If a different image repo is requested, use that with the default image name and version tag.
-	if cr.Spec.General.DefaultRepo != nil {
-		defaultRepo = *cr.Spec.General.DefaultRepo
-	}
+	//if cr.Spec.General.DefaultRepo != nil {
```

Comment:
Just remove the lines, no need to leave them in the code commented out, that just confuses people.

Reply:
Commented out code is still there